tutorial/aws-serverless-demo/SkierApiFunction/src/main/java/handlers/ResortService.java

```diff
@@ -0,0 +1,48 @@
+package handlers;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.google.gson.Gson;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import model.GatewayResponse;
+import java.util.HashMap;
+import java.util.Map;
+import model.Resort;
+import model.ResortList;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import sql.HikariDS;
+
+public class ResortService implements RequestHandler<Object, GatewayResponse> {
+  private static final Logger logger = LogManager.getLogger(ResortService.class);
```

Comment:
Context has a logger in it, so no need to instantiate one here.

Reply:
Yep I was going back and forth between using Lambda Logger or Log4j. AWS Lambda actually recommends log4j2 for a custom log appender! 

https://docs.aws.amazon.com/lambda/latest/dg/java-logging.html